packages/charts/src/core/common/scales/domain-calculation/merge-domains.spec.ts

```diff
@@ -17,10 +17,17 @@ describe("merge domain >", () => {
     it("should skip domains that are equal to EMPTY_CONTINUOUS_DOMAIN", () => {
         const scale = new LinearScale();
 
-        const emptyArr = [EMPTY_CONTINUOUS_DOMAIN, EMPTY_CONTINUOUS_DOMAIN];
+        const emptyArr = [EMPTY_CONTINUOUS_DOMAIN];
         const mergedDomains = mergeDomains(emptyArr, scale);
-        expect(mergedDomains[0]).toBeUndefined();
-        expect(mergedDomains[1]).toBeUndefined();
+        expect(mergedDomains[0]).toBeDefined();
```

Comment:
I was thinking it should be something like this:

```
const validDomain = [0, 3];
const domains = [validDomain, EMPTY_CONTINUOUS_DOMAIN];
const mergedDomains = mergeDomains(domains, scale);
expect(mergedDomains).toEqual(validDomain);
```

Reply:
Gotcha! Fixed. :)